src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -377,13 +377,14 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
             impostor.physicsBody.activate();
             var worldPoint = this._tmpAmmoVectorA;
             var impulse = this._tmpAmmoVectorB;
+            var localContactPoint = new Vector3().copyFrom(contactPoint);

```

Comment:
should not be a new here for garbage collection, would be great to use a tmp vector :-)

Reply:
(Maybe) addressed in https://github.com/BabylonJS/Babylon.js/pull/9715/commits/744a4dcdbd9e96442340b4dea40b4e3cc34ad08f